x-pack/platform/plugins/shared/task_manager/server/task_claimers/strategy_mget.ts

```diff
@@ -169,6 +171,17 @@ async function claimAvailableTasks(opts: TaskClaimerOpts): Promise<ClaimOwnershi
   const now = new Date();
   const taskUpdates: PartialConcreteTaskInstance[] = [];
   for (const task of tasksToRun) {
+    try {
+      concreteTaskInstanceSchema.validate(task);
```

Comment:
This validation currently catches the `PT1M` issue because we changed the schema for that field from `schema.duration` to `schema.string`, however prior to that fix, using this validation would not have caught the PT1M issue. I think because the goal of the issue is to catch the unknown unknown errors, we should be aiming more broadly.

Reply:
If thats the case then we could just have a massive try catch block around this entire mget claim logic, but I feel like putting more into the schema validation might be a better approach to making sure our data less error prone.

Reply:
I agree it would be cleaner but the concern is future issues where schema validation might not throw an error but the task instance is invalid due to issues outside of our control (like the `PT1M` issue).